drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -655,7 +529,7 @@ protected void getModelSuccessCallbackMethod(ScenarioSimulationModelContent cont
     }
 
     protected Command getSaveCommand() {
-        return () -> this.save("Save");
+        return () -> scenarioSimulationEditorWrapper.wrappedSave("Save");
```

Comment:
Can we use a final static constant to wrep this "SAVE" string, wdyt?

Reply:
@yesamer 
Yes - there should already be one somewhere

Reply:
Done.